python/tvm/contrib/cutlass/build.py

```diff
@@ -272,7 +274,13 @@ def handle_conv2d(
 
 
 def tune_cutlass_kernels(
-    mod, sm, use_3xtf32=True, profile_all=True, use_multiprocessing=False, tmp_dir="./tmp"
+    mod,
+    sm,
+    use_3xtf32=True,
+    profile_all_alignments=False,
+    profile_all=True,
```

Comment:
It seems to me that `profile_all_alignment` and `profile_all` may confuse people. Maybe changing to the following?
- `profile_all_alignment=False` -> `include_all_alighments=False`: Later if we have more options, we may consider having a unified `candidate_config` to include all rules.
- `profile_all=True` -> `find_first_valid=False`: This name might be clearer.

Reply:
yup agree 100% on `profile_all_alignments` vs `profile_all`, and options are getting bloated now... 

I was hesitant to rename `profile_all` because it breaks existing code (well, my code), but I'm making this change now.

Reply:
I think if we rename `profile_all` to `find_first_valid`, we can keep `profile_all_alignments`. What do you think?

Reply:
Sure I'm fine with that. It's also fine to call it `profile_configs` in the future once we want to expose more configs to users.